### ISSUE TITLE
[QoL] Processing Pattern Encoder recipe only accepts vanilla workbench, not ore dict compatible ones

### ISSUE THREAD

**Issue by kosakriszi** [raoulvdberge/refinedstorage#969]
#### Issue description:
Using ore dictionary compatible workbenches (i.e. Tinkers Construct Crafting Station) in the recipe doesn't work.

#### What happens:
The recipe is not seen as valid.

#### What you expected to happen:
To be able to use all compatible workbenches as ingredients.

#### Version:
Most likely all.

**Comment by raoulvdberge** [comment]
Define "doesn't work".

**Comment by kosakriszi** [comment]
Here's a short video detailing the issue: https://gfycat.com/UnsungRecentAntlion

**Comment by kosakriszi** [comment]
Here is a short video detailing the issue: https://gfycat.com/LamePastelCornsnake

**Comment by VT-14** [comment]
I think the recipe is using only the vanilla Crafting Table, not the ore dictionary equivalents.  Refined Storage is looking for "Blocks.CRAFTING_TABLE"

I also found Blocks.CHEST, which you could use the ore dictionary tag chestWood (I think) to add in compatibility with Quark's variants (among others).

According to that GIF, vanilla has the tags
-workbench
-craftingTableWood

While TiCon has only
-workbench

**Comment by VT-14** [comment]
I made a fork and changed the Crafting Grid, Processing Pattern Encoder, Crafting Upgrade, Disk Drive , and (in a moment, just now saw it) the Security Manager to use the ore dictionary tags "workbench" and "chestWood" as appropriate.  Unfortunately, I won't be able to test this until tonight and have no idea if I did it correctly or not (total programming experience: Introduction to C++ college class and simple Lua programs in Computercraft/OpenComputers).

I did see that the External Storage already uses the "chest" ore dictionary tag.  Is that preferred over chestWood?
